docs/experiments-rankvicuna-rerank.md

```diff
@@ -0,0 +1,74 @@
+# RankLLM: RankVicuna Re-rank Collections
+
+This guide provides instructions to reproduce the RankVicuana Re-rank model described in the following paper.  
+
+Note, we recommend using >24GB RAM for top-20 ranking, and >30GB RAM for top-100 ranking experiments.  
+Additionally, we recommend using A100 with high-RAM option when working on Colab.  
+The expected runtime for the experiments is around 3-4 mins for top-20 ranking, and 30-45 mins for top-100 ranking.  
```

Comment:
Is this for each of dl19/dl29

Reply:
The time is for each dl19/dl20, I can specify further in the doc.